### ISSUE TITLE
Potential Exception in ClassificationUnitCasMultiplier

### ISSUE THREAD

**Issue by mwunderlich** [dkpro/dkpro-tc#272]
When a CAS which does not have any "TextClassificationUnit" annotations reaches the ClassificationUnitCasMultiplier in Unit mode, the following line will throw an Exception: 

``` java
TextClassificationUnit unit = JCasUtil.selectCovered(copyJCas,  TextClassificationUnit.class, focus).get(0);
```

This can easily be avoided by checkin in the proces() method whether or not the TC unit iterator has any elements.

**Comment by daxenberger** [comment]
This was closed by commit 1881545ead548bcdf3fd587f96ddfc1edc46f9d7.

**Comment by mwunderlich** [comment]
Ah, yes, sorry. I thought it would be closed automatically by the commit message with issue ID. Maybe you need the '#' char for that to work.
